### DISCUSSION TITLE
Distro version choosing and vscode instalation

### DISCUSSION THREAD

**Discussion by rodhis** [microsoft/WSL#9538] [Q&A]
Hello! I just got into WSL2 and I have questions.

First of all, these are the distros available to use via WSL2:
22.04.1 LTS (default?): release 03/14/2022
20.04.5 LTS: release 02/23/2022
22.04.1 LTS: release 10/25/2021
18.04.5 LTS: release 05/10/2022

Should I just go with default or use something else for any reason?

Second, how should I install, VSCode? Should I use a package and install the linux version via the wsl terminal? Or should I just use the windows version and set wsl as terminal?

Thank you!

**Comment by seths10** [comment]
The default distro in WSL2 is Ubuntu 20.04 LTS, which is a stable and widely used version. However, you can choose to use any of the other distros if they better suit your needs or if you have a preference for them.

Regarding VSCode, you have two options:

You can install the Linux version of VSCode via the WSL terminal. This will give you the full Linux experience, and you can use the terminal within VSCode to run Linux commands.

You can install the Windows version of VSCode and set WSL as the terminal. This will allow you to use the Windows version of VSCode but run Linux commands in the integrated terminal.

Both options will work, it depends on your preference and the experience you want to have.

**Reply by seths10** [reply]
give me an upvote if you like the answer
